projects/vdk-core/src/vdk/internal/builtin_plugins/config/log_config.py

```diff
@@ -77,8 +78,8 @@ def configure_loggers(
             what_happened=f"Provided configuration variable for {SYSLOG_SOCK_TYPE} has invalid value.",
             why_it_happened=f"VDK was run with {SYSLOG_SOCK_TYPE}={syslog_sock_type}, however {syslog_sock_type} is invalid value for this variable.",
             consequences=errors.MSG_CONSEQUENCE_DELEGATING_TO_CALLER__LIKELY_EXECUTION_FAILURE,
-            countermeasures=f"Provide either valid value for {SYSLOG_SOCK_TYPE} or install database plugin that supports this type. "
-            f"Currently possible values are {SYSLOG_SOCK_TYPE_VALUES_DICT.keys()}",
+            countermeasures=f"Please provide a valid value for {SYSLOG_SOCK_TYPE}."
```

Comment:
nit: please is unnecessary. It was written in best practices for documentation somewhere. That this kind of "filler" words should be avoided.